### ISSUE TITLE
About the training protocol of SVHN2MNIST experiment

### ISSUE THREAD

**Issue by thudzj** [Mid-Push/Moving-Semantic-Transfer-Network#4]
I notice that ADDA(https://github.com/erictzeng/adda/blob/master/scripts/svhn-mnist.sh) use the training set of SVHN and MNIST for adaptation, but you use test set of MNIST as target domain in the code(https://github.com/Mid-Push/Moving-Semantic-Transfer-Network/blob/master/digit-dataset/smtrain.py#L39). So what do you mean by saying "follow the protocols in (Tzeng et al., 2017)" in the paper?

**Comment by Mid-Push** [comment]
Thanks for your reminder!  We checked the code of ADDA and they indeed use the training set of SVHN and MNIST.  While we used the test set in the target domain.  It may be a bit incorrect to use the "follow protocols in (Tzeng et al.)".

But we also want to note that actually more data in the target domain could further promote the  performance of adaptation. We re-run the experiments with both training split in the SVHN and MNIST. We found that our method performance increased from 91.7 to 92.5  (U can easily verify that by changing the 'test' to 'train' in line 39 and 40). Thus it is still fair to compare with Tzeng et al. and other methods.
